libqtile/config.py

```diff
@@ -483,6 +484,35 @@ def __repr__(self):
         return '<config.Group %r (%s)>' % (self.name, attrs)
 
 
+class ScratchPad(Group):
+    """Represents a "ScratchPad" group
+
+    ScratchPad adds a (by default) invisible group to qtile.
+    That group is used as place for currently not shown windows spawned by a
+    ``DropDown`` configuration.
+
+    Parameters
+    ==========
+    name : string
+        the name of this group
+    dropdowns : default ``None``
+        list of DropDown objects
+    position : int
+        group position
+    label : string
+        The display name of the ScratchPad group. Defaults to the empty string
+        such that the group is hidden in ``GroupList`` widget.
+    """
+    def __init__(self, name, dropdowns=None, position=MAXSIZE, label=''):
+        Group.__init__(self, name, layout='floating', layouts=['floating'],
+                       init=False, position=position, label=label)
+        self.dropdowns = dropdowns
+
+    def __repr__(self):
+        return '<config.ScratchPad %r (%s)>' % (
+            self.name, ', '.join(dd.name for dd in self.dropdowns))
```

Comment:
You should probably set `dropdowns` to an empty list in the `__init__` method if it is left as the default `None`